staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -91,6 +92,27 @@ type Config struct {
 	StorageObjectCountTracker flowcontrolrequest.StorageObjectCountTracker
 }
 
+// ConfigForResource is a Config specialized to a particular `schema.GroupResource`
+type ConfigForResource struct {
+	// Config is the resource-independent configuration
+	Config
+
+	// GroupResource is the relevant one
+	GroupResource schema.GroupResource
+
+	// GroupResourceString is `GroupResource.String()`
+	GroupResourceString string
```

Comment:
Let's remove this.
For the purpose of not calling string in the metrics you're adding in a subsequent PR, we can cache the string-ified group-resource in the cacher/etcd (they are not storing storagebackend.Config anyway so we can compute it there). So we don't need it in the config.

Reply:
Done